### PR TITLE
PCHR-3703: New page for Managing HR Resources

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR, a New Page for Managing HR Resources has been added and styled.

## How it looks?
**Desktop**
d

**Mobile**
m1
m2

### CODE REVIEW

File: civihr_employee_portal/views/templates/views-view--hr-resource-types-list--page.tpl.php [hunk 1]
```diff
@@ -94,4 +94,3 @@
 <?php endif; ?>
 
   </div><?php /* class view */ ?>
-
```
Comment (reneolivo) [inline civihr_employee_portal/views/templates/views-view--hr-resource-types-list--page.tpl.php]: Why was the line at the end of the file removed? I thought this was a good practice.

File: civihr_employee_portal/views/templates/views-view--manage-hr-resources-page.tpl.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+ * - $empty: The empty text to display if the view is empty
+ * - $pager: The pager next/prev links to display, if any
+ * - $exposed: Exposed widget form/info to display
+ * - $feed_icon: Feed icon to display, if any
+ * - $more: A link to view more, if any
+ *
+ * @ingroup views_templates
+ */
+?>
+<div class="<?php print $classes; ?>">
```
Comment (reneolivo) [inline civihr_employee_portal/views/templates/views-view--manage-hr-resources-page.tpl.php]: aren't you able to use smarty here? `<div class="{{ $classes }}">` etc.

Comment (hitesh-jain) [reply civihr_employee_portal/views/templates/views-view--manage-hr-resources-page.tpl.php]: @reneolivo not recommended here.

File: civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc [hunk 3]
```diff
@@ -10,6 +10,20 @@
 function civihr_employee_portal_features_views_default_views() {
   $export = array();
 
+  addLeaveAndAbsenceReportView($export);
```
Comment (hitesh-jain) [inline civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc]: @deb1990 why this functions are added ?

Comment (deb1990) [reply civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc]: > Refatored civihr_employee_portal_features.views_default.inc and moved addition of views to separate functions.

These are added to better manage the views. Previously in the same function, we were adding 4 views, and it was really tough to manage.

File: civihr_employee_portal/views/templates/views-view--hr-resource-types-list--page.tpl.php (hunk 1 above)
Comment (deb1990) [reply civihr_employee_portal/views/templates/views-view--hr-resource-types-list--page.tpl.php]: Yeah it was removed by mistake, thanks for noticing.

File: civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc (hunk 3 above)
Comment (hitesh-jain) [reply civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.views_default.inc]: @deb1990 we should avoid adding/changing feature files directly, adding other functions are not recommended.

**Pull request merged**